CourseManager/Views/ExampleViews/MatchedGeometryEffect/Views/SofaListView.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  SofaListView.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-12-08.
+//
+
+import SwiftUI
+
+struct SofaListView: View {
+    @Namespace var namespace
+    @State var selectedID = UUID()
+    @State var show = false
+    var body: some View {
+        ZStack {
+            ScrollView {
+                LazyVGrid(columns: [GridItem(.adaptive(minimum: 300), spacing: 20)], spacing: 20) {
```

Comment:
May be it is better to put the whole grid into `if` condition, because you don't need empty grid.

Reply:
Agree, updated